tutorials/monitoring/demo/walkthrough-1.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Monitoring NServiceBus Demo - Slow handlers"
+reviewed: 2017-11-03
+summary: Measuring individual endpoint performance with the throughput and processing time metrics.
+---
+
+_Which message types are taking the longest to process?_
+
+This is the first question that you should be asking when you start monitoring an NServiceBus system. The handlers for these message types are the slowest part of your system and if you want to spend time optimizing code, these message handlers are the best candidates to see improvements in your system.
+
+This part of the tutorial guides you through finding the message types that take the longest to process.
+
+include: walkthrough-solution
+
+
+## Metrics
```

Comment:
I think that there is an assumption here that we want to optimize the throughput of the system and the best way to do it is by optimizing processing time for right message types.

Is that correct? Should we make that explicit?

In such case we could change the intro a bit and say: "Let's step back and asks why are you even asking `Which message ...`". That would enable us to explain right away what is the reason we talk about Throughput and Processing time in tandem.